### PR TITLE
Fix/eco-2999

### PR DESCRIPTION
**Pull request by abhishek305**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=K8dzSYb)

Changes:

- snyk fixes
- license updated
- package version bump
- generate dot env file

### CODE REVIEW

**Comment by MaitriGurey1** [comment]
Hi @abhishek305 you have deleted the content schema from the code, but not provided any new file ? is it still working ?

**Review by surajcontentstack — APPROVED** [review]

**Comment by abhishek305** [comment]
> Hi @abhishek305 you have deleted the content schema from the code, but not provided any new file ? is it still working ?

Yes i have removed the export folder and we'll now serve through assets link in docs @MaitriGurey1

File: package.json [hunk 1]
```diff
@@ -9,22 +9,23 @@
...
-    "@angular/compiler": "^14.1.0",
-    "@angular/core": "^14.1.0",
-    "@angular/platform-browser": "^14.1.0",
-    "@angular/platform-browser-dynamic": "^14.1.0",
+    "@angular/common": "^14.2.0",
+    "@angular/compiler": "^14.2.0",
+    "@angular/core": "^14.2.0",
+    "@angular/platform-browser": "^14.2.0",
+    "@angular/platform-browser-dynamic": "^14.2.0",
     "contentstack": "3.15.2",
```
Comment (Amitkanswal) [inline package.json:23]: pls updated this to the latest

**Review by Amitkanswal — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (abhishek305) [reply package.json]: done

**Pull request merged**